packages/@angular/cli/commands/add.ts

```diff
@@ -0,0 +1,102 @@
+import chalk from 'chalk';
+import { Command, CommandScope, Option } from '../models/command';
+import { parseOptions } from '../models/command-runner';
+import { CliConfig } from '../models/config';
+import { SchematicAvailableOptions } from '../tasks/schematic-get-options';
+
+const SilentError = require('silent-error');
+
+
+export default class AddCommand extends Command {
+  readonly name = 'add';
+  readonly description = 'Add support for a library to your project.';
+  scope = CommandScope.inProject;
+  arguments = ['collection'];
+  options: Option[] = [];
+
+  private _rawArgs: any;
+
+  async initializeRaw(args: any): Promise<any> {
```

Comment:
Why not perform this in `Command`?
`private _rawArgs` => `protected rawArgs`
This would reduce the efforts needed by each command.

Reply:
Done.